orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/pipeline/PipelineStage.groovy

```diff
@@ -40,4 +42,23 @@ class PipelineStage extends LinearStage {
       buildStep(stage, "monitorPipeline", MonitorPipelineTask)
     ]
   }
+
+  @Override
+  Stage prepareStageForRestart(Stage stage) {
+    stage = super.prepareStageForRestart(stage)
+    stage.startTime = null
+    stage.endTime = null
+
+    stage.context.remove("status")
+    stage.context.remove("executionName")
+    stage.context.remove("executionId")
+
+    stage.tasks.each { com.netflix.spinnaker.orca.pipeline.model.Task task ->
```

Comment:
This should probably just be:
```groovy
stage.tasks.each { Task task ->
```
with `com.netflix.spinnaker.orca.pipeline.model.Task` as an `import`